src/chainparams.cpp

```diff
@@ -432,6 +432,10 @@ class CCustomParams : public CRegTestParams {
         // No subsidy for custom chains by default
         consensus.genesis_subsidy = args.GetArg("-con_blocksubsidy", 0);
 
+        // Note: This global is needed to avoid circular dependency
```

Comment:
?

Reply:
with chainparams (ie if the global was a field in chainparams), because chainparams depends on block.